pkg/services/system_intakes_test.go

```diff
@@ -486,22 +486,6 @@ func (s ServicesTestSuite) TestNewSubmitSystemIntake() {
 		s.Equal(&models.SystemIntake{}, intake)
 	})
 
-	s.Run("returns notification error when submit email fails", func() {
```

Comment:
Non-blocking suggestion: You could keep the majority of this test in, which would exercise the code path, just change it to show that it ends up as a success, even when the email sending errors out.